module/VuFind/src/VuFind/Captcha/Figlet.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace VuFind\Captcha;
+
+use Laminas\Mvc\Controller\Plugin\Params;
+
+class Figlet extends AbstractBase {
+    
+    protected $captcha;
+    
+    public function __construct() {
+        $this->captcha = new \Laminas\Captcha\Figlet([
```

Comment:
Might it make sense to build the `\Laminas\Captcha\Figlet` object in a factory so it's easier to customize?

Reply:
Done + moved parts of the options to config.ini

Reply:
Thanks!